bdsf/_version.py

```diff
@@ -4,14 +4,30 @@
 """
 
 # Version number
-__version__ = '1.10.1'
+__version__ = '1.10.2'
 
 
 # Changelog
 def changelog():
     """
     PyBDSF Changelog.
     -----------------------------------------------------------------------
+    2023/02/10 - Version 1.10.2
+
+    2023/02/10 - Fix issues with numpy versions >= 1.24 (#193)
```

Comment:
I just realized it was issue #192, not #193 as I wrote in the PR. Can you change it here and in `whats_new.rst`?

Reply:
Actually, #193 was a reference to the PR, so it is fine here.